lib/net/pool.js

```diff
@@ -2012,6 +2130,157 @@ class Pool extends EventEmitter {
     peer.sendCFCheckpt(packet.filterType, packet.stopHash, filterHeaders);
   }
 
+  /**
```

Comment:
Let's remove this handler as well

Reply:
Again, I think just for uniformity, lets keep it.

Reply:
removed